packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -554,7 +554,9 @@ export class AssetGraphBuilder {
                           this.options.projectRoot,
                           loc?.filePath,
                         ) ?? undefined,
-                      language: assetNode.value.type,
+                      language:
+                        incomingDep.value.sourceAssetType ??
+                        assetNode.value.type,
```

Comment:
`assetNode.value.type` is never correct, so this should rather be
```suggestion
                      language: incomingDep.value.sourceAssetType ?? undefined,
```

Reply:
OK! Thank you for helping 👍